File: tools/bench/polyglot/nim/list_sum.nim
{.push overflowChecks: off.}
var n = 1_000_000
var xs: seq[int64] = @[]
var i: int64 = 0
while i < n:
  xs.add(i)
  i = i + 1
var s: int64 = 0
var j = 0
while j < xs.len:
  s = s + xs[j]
  j = j + 1
echo s

File: tools/bench/polyglot/go/list_sum.go
package main

import "fmt"

func main() {
	const n = 1_000_000
	xs := make([]int64, 0, n)
	var i int64 = 0
	for i < int64(n) {
		xs = append(xs, i)
		i++
	}
	var s int64 = 0
	j := 0
	for j < len(xs) {
		s += xs[j]
		j++
	}
	fmt.Println(s)
}
